source/replay/replay.js

```diff
@@ -99,10 +99,8 @@ function replay(compare, driver, task, actions, browserName, HuxleyfileContainer
         console.log((err.message + ' ' + err.diffPath + '\n').red);
       } else if (err.message.indexOf('Images not the same dimension') > -1) {
         console.log((err.message.red));
-      } else {
-        // unrecognized. Throw again to have the stack trace
-        throw err;
       }
+      throw err;
```

Comment:
Thanks. What's the reasoning here though? Is it really worthwhile to rethrow when the specific errors above are already descriptive enough (I presume they are?)